addons/account_peppol/wizard/peppol_registration_views.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="peppol_registration_form" model="ir.ui.view">
+        <field name="name">peppol.registration.form</field>
+        <field name="model">peppol.registration</field>
+        <field name="groups_id" eval="[Command.link(ref('account.group_account_manager'))]"/>
+        <field name="arch" type="xml">
+            <form>
+                <field name="company_id" invisible="1"/>
```

Comment:
There is a weird padding on top... maybe it's due to a missing `sheet`? :thinking: